### PR TITLE
Calo TDAQ changes

### PR DESCRIPTION
**Pull request by sophiemiddleton**
Draft PR for changes to CaloFragments class

### CODE REVIEW

**Comment by sophiemiddleton** [comment]
It requires https://github.com/Mu2e/artdaq_core_mu2e/pull/13

**Comment by brownd1978** [comment]
Can you please relaunch the CI with the dependency?  See the instructions [here](https://mu2ewiki.fnal.gov/wiki/GitHubWorkflow#GitHub_Pull_Request_Procedures_and_FNALbuild)

**Comment by kutschke** [comment]
@sophiemiddleton  Have you requested a tag of artdaq_core_mu2e from it's authors?  That may also require tags of lower level repos.  Please learn what repos are needed and let us know.  Then we will install them in /cvmfs and make a test envset.

**Comment by sophiemiddleton** [comment]
@FNALbuild run build test with Mu2e/artdaq_core_mu2e#13

**Comment by sophiemiddleton** [comment]
@FNALbuild run build test with https://github.com/Mu2e/artdaq_core_mu2e/pull/13

**Comment by kutschke** [comment]
@FNALbuild run build test

**Comment by kutschke** [comment]
forgot we needed an updated Muse.  Thanks Ray. This time for sure.
@FNALbuild run build test

**Comment by kutschke** [comment]
There is still one more instance of "Readout" in  CaloDAQUtilities.hh .  The other changes you need to make are 

1. Remove references to  'DTCInterface' in 3 SConscript files, , in Analyses, CaloDiag and DAQ
2. In ArtBinaryPacketsFromDigis_module.cc change the include of dtcInterfaceLib/DTC_Packets.h to get the file from its new location in artdaq_core_mu2e.

And take a look at /exp/mu2e/app/users/kutschke/Bugs/PR1122/build.log.7 to see the remaining errors in the build.

One other things.  Look at UPS active.  Do you have version of mu2e_pcie_utils set up?  if so, what version is it?  You should not have it setup.  Also, what version of Muse are you using - check this with echo $MUSE_DIR .

**Comment by kutschke** [comment]
@FNALbuild run build test

**Comment by kutschke** [comment]
This PR is now next in the queue.  p051 has been updated to include both the new KinKal and the new artdaq_core_mu2e.   You need to resolve a conflict: both you have a previous PR have changed .muse.  It is now p053 in the head but you need to change it to p051.  The easiest way is to  use the web editor in GitHub - do you haver permission to do that? 

Let me know how you want to proceed.

File: .muse [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # prefer to build with this environment
-ENVSET p050
+ENVSET p053
```
Comment (kutschke) [inline .muse:2]: Please put it back to p051.  There is an email thread with the sequence we are using.  Your step is p051.  The step after that will be p053, which adds new the new art, that has yet to be validated.  We want to validate it with your PR merged in.

**Review by kutschke — CHANGES_REQUESTED** [review]

File: .muse (hunk 1 above)
Comment (sophiemiddleton) [reply .muse:2]: ok, so I dont need to change anything once I've done that?

**Review by kutschke — COMMENTED** [review]
In the end I had to use the web editor to resolve the last conflict.  Ready to retry the CI.

**Comment by kutschke** [comment]
@sophiemiddleton Please deal with the whitespace errors found by the CI and commit.

**Comment by sophiemiddleton** [comment]
It seems I had some issues in my local branch, I have tried to do this on the GUI

**Comment by kutschke** [comment]
@FNALbuild run build test

**Review by kutschke — APPROVED** [review]
Sophie is having trouble committing white space changes.  Possibly because of the web edit that I did.  I will merge this PR and make a new one to fix the whitespace errors.

**Pull request merged**